packages/lib/src/interactions/svg/SvgRect.tsx

```diff
@@ -11,22 +12,20 @@ interface Props extends SVGProps<SVGRectElement> {
 function SvgRect(props: Props) {
   const { coords, strokePosition, ...svgProps } = props;
 
-  const [start, end] = coords;
   const { stroke, strokeWidth = 1 } = svgProps;
 
   // Shrink/grow rectangle to simulate positioning stroke inside/outside
   // https://stackoverflow.com/questions/7241393/can-you-control-how-an-svgs-stroke-width-is-drawn
   const offset =
     stroke && strokePosition
-      ? (strokeWidth / 2) * (strokePosition === 'inside' ? 1 : -1)
+      ? strokeWidth * (strokePosition === 'outside' ? 1 : -1)
       : 0;
 
+  const { min, max } = Box.fromPoints(...coords).expandBySize(offset, offset);
+
   return (
-    <rect
-      x={Math.min(start.x, end.x) + offset}
-      y={Math.min(start.y, end.y) + offset}
-      width={Math.abs(end.x - start.x) - offset * 2}
-      height={Math.abs(end.y - start.y) - offset * 2}
+    <path
+      d={`M ${min.x},${min.y} H ${max.x} V ${max.y} H ${min.x} z`}
```

Comment:
Now, even if the box is smaller than its stroke, the browser still renders a line.